nextflow_schema.json

```diff
@@ -1080,6 +1066,27 @@
             },
             "required": ["report_file", "logo_file", "css_file"]
         },
+        "advanced_options": {
```

Comment:
Sorry, I think farming these off to an advanced section is unhelpful. I'd rather they were left where they were but marked as advanced and likely to not be changed.

Reply:
Aah, got it! So then we do not need an advanced section at all? Or did you have something else in mind here? https://github.com/nf-core/differentialabundance/issues/202#issuecomment-1855831683

Reply:
No, you got it right, I'm just being fickle. I don't think it actually works very well. Leaving options with their 'friends' but marking them as advanced makes more sense. (sorry ;-) )